src/platform/index.ts

```diff
@@ -114,16 +114,6 @@ export const updateOrInsertModel = (
 
 // Hook helpers
 
-export const getCurrentRequestAffectedIds: typeof sbvrUtils.getAffectedIds = (
-	args,
-) => {
-	// We store the affected ids in the custom props so we only have to fetch it once per request
```

Comment:
Don't we still need to do something similar in pinejs, like in
See: https://github.com/balena-io/pinejs/pull/379
?